packages/graphics/src/utils/index.ts

```diff
@@ -47,14 +48,14 @@ export const FILL_COMMANDS: Record<SHAPES, IShapeBuildCommand> = {
  * Batch pool, stores unused batches for preventing allocations.
  *
  * @memberof PIXI.graphicsUtils
- * @type {Array<PIXI.graphicsUtils.BatchPart>}
+ * @member {Array<PIXI.graphicsUtils.BatchPart>} BATCH_POOL
```

Comment:
I think renaming the documentation here is unnecessary but works :)

Reply:
JSDoc has issues with ES6 exports 😦 This was a case where it inferred the name as `exports.BATCH_POOL` instead of just `BATCH_POOL`. Having a better doc parser would help or, what I've seen in other places as a good stylistic choice, is don't do `export class`, `export let`, etc as inline, and do your exports at the bottom of the document `export { BATCH_POOL }`. It's easier to see exports all in one place.